deploy/converged/cluster_role.yaml

```diff
@@ -684,3 +684,69 @@ rules:
   - '*'
   verbs:
   - '*'
+
```

Comment:
If you simply added these changes to the appropriate `templates/*.yaml.in` then at least this wouldn't get blown away by another manifest generation.

Reply:
thanks for the info, moving changes there

Reply:
These things are under animated discussion recently and quickly changing. @djzager , can you please have a look and provide feedback/implementation?

Reply:
fyi., implementation of web-ui-operator components is wip now